### DISCUSSION TITLE
How to plot negative axis in Paraview

### DISCUSSION THREAD

**Discussion by salaudeen-ya** [idaholab/moose#22609] [Q&A Meshing] [answered]
Hello,

My question is:

+ Consider the attached input file and the paraview result, how do i get the result to plot from [-1,1] instead of the attached output result of [0,2] that paraview gave me?

----Paraview result plotting [0,2]-----------------
![axis-issue](https://user-images.githubusercontent.com/38023107/200371683-2c159417-5163-4630-8957-2065bf87c1da.png)

```
---------Desired output is over [-1,1]----------------
[Mesh]
  type = GeneratedMesh
  dim = 1
  nx = 64
  xmin = -1
  xmax = 1

[]

[Variables]
  [v]
    order = FIRST
    family = LAGRANGE
  []
[]

[AuxVariables]
  [flux_v]
    order = CONSTANT
    family = MONOMIAL
  []
[]

[Kernels]
  [diffusion]
    type = MatDiffusion
    variable = v
    diffusivity = 1
  []
[]

[AuxKernels]
  [fluxKernel]
    type = DiffusionFluxAux
    diffusivity = 1
    variable = flux_v
    diffusion_variable = v
    component = x
  []
[]

[BCs]
   [leftBC] 
    type = DirichletBC
    variable = v
    boundary = 'left'
    value = 0
  []

  [rightBC] 
    type = NeumannBC
    variable = v
    boundary = 'right'
    value = 3
  []
[]


[Executioner]
  type = Steady
  solve_type = 'PJFNK'
[]

[Outputs]
  exodus = true
  #csv = true
  console = true
[]

```

------------------
Thank you!

**Comment by GiudGiud** [comment, marked as answer]
Hello

This is more of a paraview topic than MOOSE.

The axis settings in paraview may be modified, the coordinates may be transformed as well, there isnt a single solution
On the left properties panel when you have the plot up, you may select a different X-axis, which could be data modified in any way you want. make sure the 'use index as x-axis' in not checked

Guillaume
